app/tests/test_app.py

```diff
@@ -625,8 +625,8 @@ def test_report_user(self):
             kwargs={'category_slug': 'articles', 'slug': self.post.slug}))
         self.assertContains(
             resp,
-            'This comment has been reported by the community and '
-            'the user has been banned')
+            'This comment has been removed by a moderator and '
+            'the user has been banned from commenting for 3 days')
```

Comment:
This should be `Your comment has been flagged by a community member. You will not be able to comment again until tomorrow.`